certified-connectors/Connect2All/apiDefinition.swagger.json

```diff
@@ -2,6 +2,23 @@
   "basePath": "/",
   "consumes": [],
   "definitions": {
+    "Actiontype": {
+      "description": "The Actiontype of the synchronous action that you wish to execute",
```

Comment:
@GAC-AJO same here. Please fix description, empty title and x-ms-summary.

Reply:
Changed Actiontype to action type.
Added title.